sql/core/src/test/resources/sql-tests/results/ansi/cast.sql.out

```diff
@@ -448,6 +496,9 @@ struct<>
 -- !query output
 java.lang.NumberFormatException
 invalid input syntax for type numeric: '1中文'. To return NULL instead, use 'try_cast'. If necessary set spark.sql.ansi.enabled to false to bypass this error.
+== SQL(line 1, position 7) ==
+select cast('1中文' as tinyint)
+       ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It's minor but seems the stop index is incorrect if the expression contains unicode.